client-src/elements/chromedash-feature-table.js

```diff
@@ -181,7 +181,7 @@ class ChromedashFeatureTable extends LitElement {
   renderPagination() {
     // Indexes of first and last items shown in one-based counting.
     const firstShown = this.start + 1;
-    const lastShown = this.start + this.features.length;
+    const lastShown = this.features.length ? this.start + this.features : firstShown;
```

Comment:
For the case where there are zero results, it might be better to not show the pagination at all.  That could be added as a new condition just before line 205:
`if (this.features.length === 0) { return nothing; }`